feat: add not-found page

floraSimpleDev opened the pull request:
adding not-found page in beequant-turbo.

Resolve CP-110

<!-- link to tickets -->

---

## What happens here?

## How do I know this is working?

## Any notes?

<!--
  New PR Checklist

- Have you tag the right reviewer?
- Have you dm the reviewer?
- Maybe @ the reviewer in channel just to be safe

- is CI passing in your pr?
- Have you update the ticket with the right status?
-->

Commit:
refactor login form

Commit:
style login form as per style guide

Commit:
flip user loading state in platform app

Commit:
more styling changes to login form

Commit:
better lookking theme toggle

Commit:
style register form

Commit:
fix input focused style

s3713434 committed:
feat: Optimize checkbox and add background colors to Tailwind config

Fixed the bug where the label and checkbox were not changing color simultaneously, added predefined background colors to the Tailwind config, and set the font size to 13px to match the previous frontend framework.

Resolve CP-97

Commit:
add more types to login form

s3713434 committed:
feat: Adding third-party section on the Login Page

Added a third-party section(buttons)
- implemented third-party section

Resolve CP-100

Commit:
merge in latest platform_app

Commit:
fix login form field not linked correctly

Commit:
add new icons for dashboard menu items

Commit:
add dashboard layout

Commit:
break out sidebar data n stuffs

Commit:
organize sidebar menu item styles

Commit:
add mobile compatibility to sidebar

Commit:
fix sidebar not overflowing when in md viewport

Commit:
add logout action in auth module

Commit:
connect dashboard link in sidebar

Commit:
fix dashboard layout content styling

Commit:
add profile menu

Commit:
more fix to dashboard layout styling

s3713434 committed:
Fixed the text-size problem

Commit:
add pull request template

Commit:
discard old apps from repo

Commit:
fix ci errors

Commit:
update ci triggers

Commit:
add turbo cache to ci

Commit:
add pr checklist

Commit:
regen yarn.lock

s3713434 committed:
Change a bit style

s3713434 committed:
Change the button color

s3713434 committed:
Update: Create Social media button

s3713434 committed:
Added primary and accent color palettes, and subsequently updated the styles for the login form, register form, buttons, checkboxes, and layout.

Commit:
Merge pull request #3 from BeeQuantAI/feat/dashboard-layout

feat: migrate dashboard layout

s3713434 committed:
Merge branch 'feat/CP-100-adding-third-party-section' into CP-102-optimize-login-and-register-form

s3713434 committed:
change the color(?

s3713434 committed:
Merge branch 'dev' into CP-102-optimize-login-and-register-form

s3713434 committed:
Change the dashboard color(?

s3713434 committed:
Update: Solved most of problems, will working on the color now

s3713434 committed:
adding back the text color

Commit:
prettify entire repo

Commit:
handle unrecorded auth error

Commit:
move dashboard layout stuffs around

Commit:
turn off next image warning

Commit:
add profile card

s3713434 committed:
Changed the color scale

s3713434 committed:
Merge branch 'CP-102-optimize-login-and-register-form' of https://github.com/BeeQuantAI/beequant-turbo into CP-102-optimize-login-and-register-form, and modfied the color scale

Commit:
refactor dashboard layout with new colors

Commit:
remove pgadmin from backend docker compose

Commit:
fix sidebar button hover effect

Commit:
break out tailwind colors

Commit:
Merge pull request #5 from BeeQuantAI/feat/migrate-profile-page

feat/migrate profile page

Commit:
update README.md with new instruction to start up client

Commit:
refactor social button a bit to restrict it's useage

Commit:
Merge remote-tracking branch 'origin/dev' into CP-102-optimize-login-and-register-form

Commit:
fix import error in login form

Commit:
fix props type in social button

Commit:
fix checkbox having fixed width

Commit:
more color fixes

Commit:
Merge pull request #4 from BeeQuantAI/CP-102-optimize-login-and-register-form

Cp 102 optimize login and register form

Commit:
Merge pull request #1 from BeeQuantAI/feat/CP-97-optimize-checkbox

feat: Optimize checkbox and add background colors to Tailwind config

Commit:
overhaul theming in login form

s3713434 committed:
create the 200 page

s3713434 committed:
Merge branch 'dev' of https://github.com/BeeQuantAI/beequant-turbo into feat/cp-107-create-registration-success-page

s3713434 committed:
Merge branch 'main' into feat/cp-107-create-registration-success-page

Commit:
oops

99032000 committed:
clean dependency

Commit:
able to start both client and server in prod

Commit:
downgrade next to 14.1.4

Commit:
refresh yarn.lock

Commit:
downgrade yarn to 1.22.22

Commit:
Merge remote-tracking branch 'origin/dev' into feat/theming-reimagined

Commit:
refresh yarn.lock

Commit:
add missing import in tailwind config

Commit:
refactor auth forms

Commit:
fix button having fixed width

Commit:
slow down global animation transition

Commit:
refactor controlled text input with error message

Commit:
tweak input error message color

Commit:
restyle form error message

Commit:
fix password input not hiding password

Commit:
overhaul dashboard layout styling

Commit:
restyle profile card

Commit:
Merge pull request #7 from BeeQuantAI/feat/theming-reimagined

feat/theming reimagined

Commit:
Merge remote-tracking branch 'origin/dev' into feat/cp-107-create-registration-success-page

Commit:
rerun graphql codegen in client

Commit:
add simple message about empty root page

Commit:
move theme switcher from root layout to auth layout

Commit:
move theme switcher from root layout to auth layout

s3713434 committed:
change back the register action

s3713434 committed:
Merge branch 'feat/cp-107-create-registration-success-page' of https://github.com/BeeQuantAI/beequant-turbo into feat/cp-107-create-registration-success-page

s3713434 committed:
Delete the await at auth-service.ts no need for it in redirect.

s3713434 committed:
Merge pull request #6 from BeeQuantAI/feat/cp-107-create-registration-success-page

Feat/cp 107 create registration success page

Commit:
stop tracking yarn file

Commit:
migrate identical styling from old platform app

Commit:
move pr template to a level up

Commit:
add dark mode to account-menu

Commit:
supress inconsistent attribute warning on html tag

Commit:
remove conflicting favicon file

Commit:
update accent color palette

s3713434 committed:
feat: add custom font size configuration in Tailwind CSS theme

s3713434 committed:
chore: update existing font sizes to match custom configuration in Tailwind CSS

SeijaHuang committed:
Merge branch 'dev' of https://github.com/BeeQuantAI/beequant-turbo into dev

Commit:
Feat: Optimized the README file

Optimized the README file:
- Optimized the `## How do I run this`

Resolve: Under CP-106

Chris-Wang committed:
Merge pull request #12 from BeeQuantAI/feat/Optimize-README-File

Feat: Optimized the README file

Commit:
fix sidebar menu accordion hover styles

Commit:
tweak accent palette color

Commit:
Merge pull request #11 from BeeQuantAI/cp-104-organize-and-sort-fonts-from-legacy-repo-by-size

Cp 104 organize and sort fonts from legacy repo by size

Commit:
Merge remote-tracking branch 'origin/dev' into feat/cp-111-dashboard-style-tweak

SeijaHuang committed:
Merge branch 'dev' of https://github.com/BeeQuantAI/beequant-turbo into dev

SeijaHuang committed:
feat: add setting page
Added account setting page:
- implemented account setting form
- implemented account setting page

Added variant for shared input component:
- added display variant to enable the label and input displaying either horizontally or vertically

Added user help messages and validation messages:
- added user help messages to automatically pop up when clicking responding input field
- added validation messages which are mainly for form validation error messages

Resolve CP-108

Commit:
more dashboard tweaks

Chris-Wang committed:
Merge pull request #9 from BeeQuantAI/feat/CP-108-add-setting-page

Feat/cp 108 add setting page

Commit:
hardcode line-height in sidebar menu

Commit:
regress one stop dev solution to make it compatible with windows env

Commit:
update readme with new instruction to start local dev env

Commit:
clean up readme

s3713434 committed:
refactor: match icon colors with old repo and clean up input control font sizes

Commit:
Merge branch 'dev' into feat/cp-111-dashboard-style-tweak

Commit:
Merge pull request #10 from BeeQuantAI/feat/cp-111-dashboard-style-tweak

feat/cp 111 dashboard style tweak

s3713434 committed:
refactor: convert existing register-succeed page into a reusable component

99032000 committed:
clean dependency

s3713434 committed:
feat: add responsive design to registration and login pages

floraSimpleDev committed:
fix: Replace Icons to Match Legacy Repo Styles

Fixed the Show Password Icon Animation

Resolve CP-116

fix/CP-116-Replace-Icons-to-Match-Legacy-Repo-Styles

s3713434 committed:
feat: implement ultrawide screen responsiveness for login, registration, and 200 pages

s3713434 committed:
Yeah change back the font-size

s3713434 committed:
change the min-w to max-w :3 it should be good now

SeijaHuang committed:
Feat: add internationalization
Add app setting page:
- Implemented language change
- Implemented theme change
Add internationalization
- Add next-intl library to achieve internationalization routing
- Modify middleware with next-intl middleware
- Modify next navigations with next-intl shared routing, including redirect, useRouter etc.
Resolve CP-113

floraSimpleDev committed:
feat: add not-found page

adding not-found page in beequant-turbo.

Resolve CP-110

Chris-Wang reviewed (changes requested):
Still no screenshot?

floraSimpleDev commented:
I attached the screenshots in the comments.

On Mon, 26 Aug 2024 at 7:57 PM, Chris-Wang ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Still no screenshot?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2260297535>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGDAG45CXT5QCDQJFUO3673ZTL3YTAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRQGI4TONJTGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

Chris-Wang commented:
> I attached the screenshots in the comments.
> […](#)
> On Mon, 26 Aug 2024 at 7:57 PM, Chris-Wang ***@***.***> wrote: ***@***.**** requested changes on this pull request. Still no screenshot? — Reply to this email directly, view it on GitHub <[#24 (review)](https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2260297535)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BGDAG45CXT5QCDQJFUO3673ZTL3YTAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRQGI4TONJTGU> . You are receiving this because you authored the thread.Message ID: ***@***.***>

ahh, I guess the screenshot didn't finish uploading... AND you just complete the msg

floraSimpleDev commented:
I see the link:
https://github.com/BeeQuantAI/beequant-turbo/commit/59aa1b49ee15895ad149d55f01b1a965bec2d399#comments

Chris-Wang ***@***.***> 于2024年8月26日周一 20:52写道：

> I attached the screenshots in the comments.
> … <#m_2476027636957313147_>
> On Mon, 26 Aug 2024 at 7:57 PM, Chris-Wang *@*.*> wrote: @.** requested
> changes on this pull request. Still no screenshot? — Reply to this email
> directly, view it on GitHub <#24 (review)
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2260297535>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/BGDAG45CXT5QCDQJFUO3673ZTL3YTAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRQGI4TONJTGU
> . You are receiving this because you authored the thread.Message ID: *@*
> .***>
>
> ahh, I guess the screenshot didn't finish uploading... AND you just
> complete the msg
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#issuecomment-2309919673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGDAG472FVP5IQZGNZWSZ73ZTMCGHAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMBZHEYTSNRXGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

floraSimpleDev commented:
![not-found turbo page 01](https://github.com/user-attachments/assets/35b9ae53-fb55-4e16-8abd-5193451e9130)
![not-found turbo page 02](https://github.com/user-attachments/assets/4a93a4d9-091d-460e-8c36-262d6c19a471)

Chris-Wang reviewed (changes requested):
1. The background color of 404 won't switch when theme colour changes
2. Needs to merge dev and add translation to this page

floraSimpleDev commented:
I try to find why it cannot switch theme color but failed, it seems like
couldn’t read the local storage theme value immediately.

On Tue, 27 Aug 2024 at 8:21 AM, Chris-Wang ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
>    1. The background color of 404 won't switch when theme colour changes
>    2. Needs to merge dev and add translation to this page
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2261752953>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGDAG4Z56P56H35PJAL5DLTZTOS6RAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRRG42TEOJVGM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

Chris-Wang commented:
> I try to find why it cannot switch theme color but failed, it seems like couldn’t read the local storage theme value immediately.
> […](#)
> On Tue, 27 Aug 2024 at 8:21 AM, Chris-Wang ***@***.***> wrote: ***@***.**** requested changes on this pull request. 1. The background color of 404 won't switch when theme colour changes 2. Needs to merge dev and add translation to this page — Reply to this email directly, view it on GitHub <[#24 (review)](https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2261752953)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BGDAG4Z56P56H35PJAL5DLTZTOS6RAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRRG42TEOJVGM> . You are receiving this because you commented.Message ID: ***@***.***>

1. If the PR is done without theme switching, it should be clearly stated in the description, not caught by reviewer
2. If you have such a question, why not ask?
3. If you ask a question, you should add more context, like listing the way you have tried, and how they failed, and any other effort you have put, have you google or research on that
4. We have several ways to get the theme colour in a component, I just don't know how you failed

floraSimpleDev commented:
I’m sorry about that, but I rewrite the whole not-found page code, and I
learned how others write the theme color code yesterday cause I wish to
find its operating way. And imitated others code try to fix that. But I
haven’t searched the google about it yesterday. I’m sorry.
Then I prepared to ask this question on today’s morning stand up meeting,
to find some solution ideas.

On Tue, 27 Aug 2024 at 8:41 AM, Chris-Wang ***@***.***> wrote:

> I try to find why it cannot switch theme color but failed, it seems like
> couldn’t read the local storage theme value immediately.
> … <#m_-7593328261339857620_>
> On Tue, 27 Aug 2024 at 8:21 AM, Chris-Wang *@*.*> wrote: @.** requested
> changes on this pull request. 1. The background color of 404 won't switch
> when theme colour changes 2. Needs to merge dev and add translation to this
> page — Reply to this email directly, view it on GitHub <#24 (review)
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2261752953>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/BGDAG4Z56P56H35PJAL5DLTZTOS6RAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRRG42TEOJVGM
> . You are receiving this because you commented.Message ID: *@*.***>
>
>
>    1. If the PR is done without theme switching, it should be clearly
>    stated in the description, not caught by reviewer
>    2. If you have such a question, why not ask?
>    3. If you ask a question, you should add more context, like listing
>    the way you have tried, and how they failed, and any other effort you have
>    put, have you google or research on that
>    4. We have several ways to get the theme colour in a component, I just
>    don't know how you failed
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#issuecomment-2311231457>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGDAG42TSQ27SXHIU5NPK3DZTOVH7AVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMJRGIZTCNBVG4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

floraSimpleDev commented:
I’m sorry, I will learn how to do the theme color switching today.

On Tue, 27 Aug 2024 at 8:41 AM, Chris-Wang ***@***.***> wrote:

> I try to find why it cannot switch theme color but failed, it seems like
> couldn’t read the local storage theme value immediately.
> … <#m_-7593328261339857620_>
> On Tue, 27 Aug 2024 at 8:21 AM, Chris-Wang *@*.*> wrote: @.** requested
> changes on this pull request. 1. The background color of 404 won't switch
> when theme colour changes 2. Needs to merge dev and add translation to this
> page — Reply to this email directly, view it on GitHub <#24 (review)
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#pullrequestreview-2261752953>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/BGDAG4Z56P56H35PJAL5DLTZTOS6RAVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDENRRG42TEOJVGM
> . You are receiving this because you commented.Message ID: *@*.***>
>
>
>    1. If the PR is done without theme switching, it should be clearly
>    stated in the description, not caught by reviewer
>    2. If you have such a question, why not ask?
>    3. If you ask a question, you should add more context, like listing
>    the way you have tried, and how they failed, and any other effort you have
>    put, have you google or research on that
>    4. We have several ways to get the theme colour in a component, I just
>    don't know how you failed
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BeeQuantAI/beequant-turbo/pull/24#issuecomment-2311231457>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGDAG42TSQ27SXHIU5NPK3DZTOVH7AVCNFSM6AAAAABNDKKRESVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMJRGIZTCNBVG4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

s3713434 committed:
problem solved, just misunderstanding. xp

s3713434 committed:
Merge branch 'dev' into cp-114-add-responsive-design-to-registeration-and-login-page

s3713434 committed:
Merge pull request #17 from BeeQuantAI/cp-114-add-responsive-design-to-registeration-and-login-page

feat: add responsive design to registration and login pages

Commit:
migrate theme switcher to use next-themes

SeijaHuang committed:
Feat: add account management and change password page

Add account management page：
- Implemented account management page
- Added account management menu on sidebar and account drop down
Add change password page:
- Implemented change password page with responding mutation
- Added Modal component
Relocate account setting page
- Relocate account setting page with newly added two pages in account folder which is same level with dashboard folder
Resolve CP-125

Commit:
Feat: Migrate landing page  into turbo repo

Completed the following changes:
Client:
- Add the landing page to /app named landing
- Split the landing page to eight operational components to module folder named landing
- Added the images the landing page required into public folder
- Added css files into /app/assets/css
- Imported css files into globals.css
- Changed the redirection logic in middleware.ts, if the token is not exist in the cookies, it will redirect to landing page
- Set the internationalization for landing page

Resolve CP-106

Chris-Wang reviewed (changes requested):
Needs to resolve conflict

floraSimpleDev committed:
Merge branch 'dev' into feat/CP-110-add-not-found-page and resolve conflicts

floraSimpleDev committed:
Merge branch 'dev' into feat/CP-110-add-not-found-page and resolve conflicts

floraSimpleDev committed:
Merge branch 'feat/CP-110-add-not-found-page' of https://github.com/BeeQuantAI/beequant-turbo into feat/CP-110-add-not-found-page

Commit:
fix missing next-theme setup

Commit:
clean out old theme scripts

Chris-Wang commented:
test failed:
case 1:
- after login: 
url: http://localhost:3010/zh-cn/dashboard23232
<img width="1364" alt="image" src="https://github.com/user-attachments/assets/b86c081e-6434-481e-a6f8-31a818bde5c8">
case 2:
- after login:
url: http://localhost:3010/en/appsetting23232www
<img width="1330" alt="image" src="https://github.com/user-attachments/assets/d9519bd9-f54e-4dc6-8aeb-1f4c8cceff54">
case 3:
- before login:
url: http://localhost:3010/wexzerwwer
will automatically redirect to login page, in opposite to 404 page

Chris-Wang reviewed (changes requested):
PLEASE UPDATE

floraSimpleDev committed:
fix the wrong router of not-found page
add next/intl for not-found page

floraSimpleDev commented:
![not-found turbo page 01](https://github.com/user-attachments/assets/74b42a42-72fb-40c2-b1ed-be278f06a29d)
![not-found turbo page 02](https://github.com/user-attachments/assets/2d21114e-4a48-4704-911b-c47a5f5732d2)
![not-found turbo page 03](https://github.com/user-attachments/assets/4ab155b9-da4d-4ff9-9d9e-5afdae8bb15b)
![not-found turbo page 04](https://github.com/user-attachments/assets/d4805859-46e9-47b0-9683-605885f73a76)

Chris-Wang reviewed (changes requested):
Still failed when user not sign in:
- when user not login into the system, it won't redirect user to 404 page

Commit:
use turbopack in client app

Maaayhan committed:
update: enhance zod schema validation for Login and Register pages

- update schema for the login page to ensure 'email' conforms to the standard and 'password' is not empty.
- update the schema for the registration page to ensure all fields conform to the standards and 'confirmPassword' matches 'password'.
- add multilingual support
- add passwordSchema in 'utils' for reuse

Resolved CP-126

floraSimpleDev committed:
fix not found error with pagesWithoutToken

Chris-Wang commented on .vscode/c_cpp_properties.json:
```diff
@@ -0,0 +1,18 @@
+{
```
where is this come from?

Chris-Wang commented on .vscode/launch.json:
```diff
@@ -0,0 +1,24 @@
+{
```
where is this come from?

Chris-Wang commented on .vscode/settings.json:
```diff
@@ -0,0 +1,59 @@
+{
```
where is this come from?

Chris-Wang commented on apps/client/src/module/common/not-found-page.tsx:
```diff
@@ -0,0 +1,31 @@
+            className="w-full max-w-[31.25rem]"
+          />
+          <div className="mb-5 mt-[5.625rem] text-[1.1875rem]">{message}</div>
+          <AuthRoute.Login.Link>
```
So, if a user has signed in, this button will lead him back to Login page? 
The correct behaviour should be:
- redirect to home page
- home page means:
    - landing page when user not signed in
    - dashboard/home page of platform when user signed in

Chris-Wang reviewed (changes requested):
1. Remove unused files
2. Merge the latest dev
3. Change back home button logic

Maaayhan committed:
Merge pull request #20 from BeeQuantAI/update/CP-126-enhance-zod-schema-validation

update: enhance zod schema validation for Login and Register pages

floraSimpleDev committed:
fix back home link

floraSimpleDev committed:
Merge branch 'dev' of https://github.com/BeeQuantAI/beequant-turbo into feat/CP-110-add-not-found-page

floraSimpleDev committed:
Merge branch 'dev' of https://github.com/BeeQuantAI/beequant-turbo into feat/CP-110-add-not-found-page

floraSimpleDev committed:
Merge branch 'feat/CP-110-add-not-found-page' of https://github.com/BeeQuantAI/beequant-turbo into feat/CP-110-add-not-found-page

floraSimpleDev committed:
Merge branch 'feat/CP-110-add-not-found-page' of https://github.com/BeeQuantAI/beequant-turbo into feat/CP-110-add-not-found-page

floraSimpleDev committed:
Merge branch 'feat/CP-110-add-not-found-page' of https://github.com/BeeQuantAI/beequant-turbo into feat/CP-110-add-not-found-page

floraSimpleDev committed:
fix landing page link effect

floraSimpleDev committed:
upload images and their routes

floraSimpleDev committed:
clean out extra images

Chris-Wang commented:
closed as it's been included in signed in PR
